react/orders101/src/features/BasketOpen.jsx

```diff
@@ -0,0 +1,23 @@
+function BasketOpen() {
+  function handleClick() {
+    document.querySelector("#cart-icon").addEventListener("click", function () {
```

Comment:
I think we don't need to assign the event listener in the click handler, but instead just run the callback?

Since React will assign the event listener for us when passing the `onClick` prop

Reply:
Event listener removed from the handleClick function for both BasketOpen and BasketClose